public/instructors_list.php

```diff
@@ -85,6 +111,72 @@
 
 <br><br>
 
+<h2>Courses assigned to Instructors</h2>
+<?php if ($course_assignment && count($course_assignment)) { ?>
+    <table>
+            <thead>
+                <tr>
+                    <th>Course Offering</th>
+                    <th>Instructor</th>
+                    <th>Created At</th>
+                </tr>
+            </thead>
+            <tbody>
+        <?php foreach ($course_assignment as $row) { 
+            //$inst = User::find_by(array('id' => $row->user_id));
+            //$off = CourseOffering::find_by(array('id' => $row->offering_id));
+
+            //$inst_name = $inst.get_full_name();
+            //$off_name = $off->course_offering_name;
+            ?>
+            <tr>
+                <td><?php echo $row->course_offering->course_offering_name; ?></td>
+                <td><?php echo $row->user->first_name." ".$row->user->last_name; ?></td>
```

Comment:
Heres the reason behind my comment on preloading before

Reply:
for instance, if we have this 
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/44499264/159959589-ebe2b1aa-9bca-48c5-abd4-ff6967015545.png">

we dont want 4 differnt queries to get all this, we want it to be done in 2 queries, one for the users one for the courseofferings, imagine if we have 20 course offerings on this page :) that would imply that we 'd 40 queries here and it'd be a nightmare 
![image](https://user-images.githubusercontent.com/44499264/159959639-86e5f86c-3c52-410d-80a0-92a799e86083.png)

Reply:
you dont have the logging capability in your branch yet you need to rebase from main